libs/azure-api-client/composer.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "keboola/azure-api-client",
+    "type": "library",
+    "license": "MIT",
+    "description": "Keboola Azure API client",
+    "authors": [
+        {
+            "name": "Keboola",
+            "email": "devel@keboola.com"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Keboola\\AzureApiClient\\": "src/"
+        }
+    },
+    "require": {
+        "php": "^8.1",
+        "guzzlehttp/guzzle": "^7.5",
+        "monolog/monolog": "^2.0|^3.0",
```

Comment:
to víme, že to pojede s 2 i 3?

Reply:
jo, neni tu zadna customizace, klient pouziva `Monolog\Logger` pro zapis logu